File: hello.c
//hello in C


#include <stdio.h>

int sayHello();
int sayBonjour();

int main() {

    sayHello();
    sayBonjour();

    return 0;
}

int sayHello(){

    const char* message = "Hello!";
    printf("%s\n", message);

    return 0;
}

int sayBonjour(){

    const char* message_fr = "Bonjour";
    printf("%s\n", message_fr);

    return 0;
}

// Output: Hello!
// Output: Bonjour

// End of code

File: hello.go
//hello in Go

package main
import "fmt"

func main() {
	sayHello()
	sayBonjour()
}

func sayHello(){

	message := "Hello!"
	fmt.Println(message)
}	

func sayBonjour(){

	greeting_fr := "Bonjour"
	fmt.Println(greeting_fr)
}

// Output: Hello!
// Output: Bonjour

//end of code